### PR TITLE
Lang: Chinese Simplified Translation [WIP] [Help Wanted]

### PR DESCRIPTION
**Pull request by Thunder33345**
DO NOT MERGE, it's not completed
This is here just for references, may merge if i ever became inactive
Please do help me translate as i do not have a very good translating skills

- [ ] Requesting it to be reviewed
- [ ] Requesting assist on certain parts
- [x] Make sure no traditional chinese slips in

would be nice to have multiple reviewers to proof read it or so
Link: https://github.com/Thunder33345/PiggyAuth/blob/master/resources/lang_chs.yml

### CODE REVIEW

**Comment by Thunder33345** [comment]
semi completed
DO NOT MERGE
NOT READY
requesting reviews
requesting assist (see comments)
i also need to clear out traditional chinese

**Comment by DaPigGuy** [comment]
Progress here?

**Comment by Thunder33345** [comment]
see top checkmark list
i personally dont trust myself to translate it perfectly so i would feel safer with someone to proof read it

**Comment by Aericio** [comment]
Let's oink. @MCPEPIG

File: resources/lang_chs.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+email-login-from-new-ip: "您的账户已被登录在 {ip} (IP地址). 如果那不是您,请马上更换您的密码."
+
+#Pin
+your-pin: "&a您的 pin: {pin}"
+
+#Key
+key-expired: "&c您的密钥已过期."
+
+#General
+dont-say-password: "&c请别分享您的密码." #need checking
```
Comment (DaPigGuy) [inline resources/lang_chs.yml]: According to Google Translate, it is correct (Please do not share your password)

Comment (Aericio) [reply resources/lang_chs.yml]: according to "inaccurate translate", it can be wrong.

Comment (Thunder33345) [reply resources/lang_chs.yml]: that the problem with translating over google but it now seems fine i suppose

**Comment by Thunder33345** [comment]
updated it and added the new line about "new-player"
may just push it if no one here can help, that would be the best we can do

**Comment by Aericio** [comment]
are you trying to in traditional or simplified chinese?

**Comment by Aericio** [comment]
i see some traditional chinese words that aren't simplified.

**Comment by Thunder33345** [comment]
simplifyed
also do point out all the traditional chinese words, i manage to get a few slipped in on accident, google translating it wont do the trick and i am not really sure if i can differentiate between both

File: resources/lang_chs.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# Thunder33345
+
+#Join
+join-message: "&6此服务器需要密码认真."
+new-player: "&e欢迎 {player} 来到此服务器"
+
+#Login
+login-message: "&6请登入通过 /login <密码>\n&忘记您的密码? 用 /forgotpassword <pin> <新密码>"
+login-popup: "&6请登入."
+login-tip: "&6请登入."
```
Comment (DaPigGuy) [inline resources/lang_chs.yml:13]: You are missing login-title, login-subtitle, & login-actionbar (Same as login-popup & login-tip).

File: resources/lang_chs.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+incorrect-password: "&c密码不对. 剩下 {tries} 尝试."
+not-registered: "&c您还没注册."
+too-many-tries: "&c太多登入尝试"
+timeout-message: "&c登入超时"
+too-many-on-ip: "&c太多用户有同样IP地址."
+
+#Register
+register-message: "&6请注册通过 /register <密码> <确认密码> [电邮]"
+register-popup: "&6请注册."
+register-tip: "&6请注册."
```
Comment (DaPigGuy) [inline resources/lang_chs.yml:32]: You are missing register-title, register-subtitle, & register-actionbar (Same as register-popup & register-tip).

File: resources/lang_chs.yml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+preregister-success: "&a账户成功注册."
+already-registered-two: "&c账号已被注册."
+
+#Change password
+change-password-success: "&a您密码宜被更换. 您的新 pin 是 {pin}."
+incorrect-password-other: "&c密码不对."
+
+#Forget password
+forgot-password-success: "&a您的密码已被更换. 您的 pin 是 {pin}."
+incorrect-pin: "&cpin 不对."
```
Comment (DaPigGuy) [inline resources/lang_chs.yml]: Missing {tries} left part

**Comment by Aericio** [comment]
您 -> 你. That's the one I noticed instantly that was traditional :P Think there are some others.. going to keep looking.

**Comment by Thunder33345** [comment]
well AFAIK 您 and 你 is different, they both means the same thing in english which is "you" but 您 is in a more formal and respectful manner to address a person

**Comment by Aericio** [comment]
@Thunder33345 Meh, whatever then.

**Comment by Thunder33345** [comment]
As far as things goes, you may just merge it for the time begin
i may open future PRs to fill in english lines

**Comment by Aericio** [comment]
Okay.

**Pull request merged**